dev-util/cmake/files/cmake-3.6.3-FindCUDA.patch

```diff
@@ -0,0 +1,27 @@
+--- a/Modules/FindCUDA/run_nvcc.cmake	2016-12-17 16:46:54.370587130 +0000
++++ b/Modules/FindCUDA/run_nvcc.cmake	2016-12-17 16:53:10.161588315 +0000
+@@ -104,14 +104,17 @@
+ # Add the build specific configuration flags
+ list(APPEND CUDA_NVCC_FLAGS ${CUDA_NVCC_FLAGS_${build_configuration}})
+ 
++# The following may cause an nvcc fatal: redefenition of compiler-bindir
++if(NOT @CMAKE_GENTOO_BUILD@ STREQUAL ON AND NOT @CMAKE_BUILD_TYPE@ STREQUAL Gentoo )
```

Comment:
I'm closing this. There's no point to leaving it open. It got fixed in gentoo/gentoo#3013
Thank you all